modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -7405,6 +7405,8 @@ feature.flag.LPD-24498.description=Automatically register an object action to en
 feature.flag.LPD-24498.title=Commerce Order Email Notification Template
 feature.flag.LPD-29516.description=Allows users to upload images to the blog repository even when the workflow is active.
 feature.flag.LPD-29516.title=Image Uploads to Blog Repository with Active Workflow
+feature.flag.LPD-31212.description=Users can create rich text content that includes scripting snippets.
+feature.flag.LPD-31212.title=Disable XSS validations in Rich Text Editor
```

Comment:
Hey @dsanz I think you forgot to run `gw buildlang` to generate the translations, right?

Reply:
yep, I the past I always did that step, nowadays I'm under the impression this thing happens under the hood every in a while.

I've added an extra commit